supportedBackends/aws/src/main/scala/cromwell/backend/impl/aws/AwsBatchConfiguration.scala

```diff
@@ -56,6 +55,11 @@ class AwsBatchConfiguration(val configurationDescriptor: BackendConfigurationDes
     case false =>
     PathBuilderFactory
   }
+
+  val dockerCredentials = BackendDockerConfiguration.build(configurationDescriptor.backendConfig).dockerCredentials
+  val dockerEncryptionKeyName: Option[String] = dockerCredentials flatMap { _.keyName }
+  val dockerEncryptionAuthName: Option[String] = dockerCredentials flatMap { _.authName }
+  val dockerToken: Option[String] = dockerCredentials map { _.token }
```

Comment:
Would it be possible to do the private Docker token upsert once here rather than in each AwsBatchJob? The Docker configuration remains the same during the Cromwell instance lifespan so it doesn't seem necessary to reupload that secret for every job.